pcsx2/PAD/Linux/SDL/joystick.cpp

```diff
@@ -82,12 +82,23 @@ void JoystickInfo::Rumble(unsigned type, unsigned pad)
 		return;
 
 	if (m_haptic == nullptr)
-		return;
+	{
+		// If haptic is not implemented for a controller, try to use rumble API instead
+		static const Uint16 rumble_force = SDL_MAX_UINT16;
+		static const Uint16 big_motor[NB_EFFECT] = { 0, rumble_force, };
+		static const Uint16 small_motor[NB_EFFECT] = { rumble_force, 0, };
 
-	int id = m_effects_id[type];
-	if (SDL_HapticRunEffect(m_haptic, id, 1) != 0)
+		if (SDL_GameControllerRumble(m_controller, big_motor[type], small_motor[type], s_effect_duration_ms) != 0)
+		{
+			fprintf(stderr, "ERROR: Rumble is not working! %s, type is %u\n", SDL_GetError(), type);
```

Comment:
Using Console.Error might be nicer.

Reply:
It is not used anywhere in this file. Maybe better fix it everywhere in other PR?

Reply:
I'd say change it now since you're changing the line anyway.

Reply:
Done